src/App.tsx

```diff
@@ -1,16 +1,30 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const onAdd = (newMovie: Movie) => {
```

Comment:
```suggestion
  const addMovie = (newMovie: Movie) => {
```
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/96409563/161bf6a6-78c3-46be-a5ab-66af971bb63c)